### PR TITLE
[Web Portal] Preview and edit user info before batch create users.

### PR DESCRIPTION
**Pull request by mslichao**
#2491 and #2492 in #2078 

Teacher want to batch create/update/delete user accounts for his class.

This PR is preview and edit user info before batch create users, the whole steps is in #2078 .

image

image

### CODE REVIEW

**Comment by Gerhut** [comment]
Please use PascalCase in filename if it only contains a Component / Context (due to it must be used as `Context.Provider` / `Context.Consumer` in JSX)

**Comment by mslichao** [comment]
Thanks for your suggestion, fixed.

---
In reply to: [486573961](https://github.com/Microsoft/pai/pull/2665#issuecomment-486573961) [](ancestors = 486573961)

File: src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+function TopBar() {
+  const {importFromCSV, downloadTemplate, addNew} = useContext(Context);
+
+  const buttonPrimaryStyles = {margin: '0 1rem 0 -1rem'};
+  const buttonDefaultStyles = {margin: '0 1rem', backgroundColor: '#DADADA'};
+
+  /**
+   * @returns {import('office-ui-fabric-react').ICommandBarItemProps}
+   */
+  function getBrowse() {
```
Comment (sunqinzheng) [inline src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx]: Why a function is used here?
I think it just returned a static object.

File: src/webportal/src/app/user/fabric/batchRegister/Table.jsx [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconContainerStyle = {marginTop: 8, marginBottom: 8, marginLeft: 8};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconStyle = {color: 'white'};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const textStyle = {marginTop: 8, marginRight: 8, marginBottom: 8, color: 'white'};
+      return (
+        <div style={Object.assign(wrapperStyle)}>
+          <TooltipHost content={message}>
+            <MessageBar
```
Comment (sunqinzheng) [inline src/webportal/src/app/user/fabric/batchRegister/Table.jsx]: Please use the statusBadge component in src/app/components. (You may need rebase/merge your branch to master...)

File: src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx (hunk 1 above)
Comment (mslichao) [reply src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx]: Good suggestion, fixed.

File: src/webportal/src/app/user/fabric/batchRegister/Table.jsx (hunk 2 above)
Comment (mslichao) [reply src/webportal/src/app/user/fabric/batchRegister/Table.jsx]: Rebased and used statusBadge, thanks for your suggestion.

**Pull request merged**